src/BridgeHelper/ExternalDictionaryLibraryBridgeHelper.h

```diff
@@ -74,18 +76,21 @@ class LibraryBridgeHelper : public IBridgeHelper
 
     Poco::URI createBaseURI() const override;
 
+    QueryPipeline loadBase(const Poco::URI & uri, ReadWriteBufferFromHTTP::OutStreamCallback out_stream_callback = {});
+
+    bool executeRequest(const Poco::URI & uri, ReadWriteBufferFromHTTP::OutStreamCallback out_stream_callback = {}) const;
+
     ReadWriteBufferFromHTTP::OutStreamCallback getInitLibraryCallback() const;
 
 private:
-    static constexpr inline auto LIB_NEW_METHOD = "libNew";
-    static constexpr inline auto LIB_CLONE_METHOD = "libClone";
-    static constexpr inline auto LIB_DELETE_METHOD = "libDelete";
-    static constexpr inline auto LOAD_ALL_METHOD = "loadAll";
-    static constexpr inline auto LOAD_IDS_METHOD = "loadIds";
-    static constexpr inline auto LOAD_KEYS_METHOD = "loadKeys";
-    static constexpr inline auto IS_MODIFIED_METHOD = "isModified";
-    static constexpr inline auto PING = "ping";
-    static constexpr inline auto SUPPORTS_SELECTIVE_LOAD_METHOD = "supportsSelectiveLoad";
+    static constexpr inline auto EXT_DICT_LIB_NEW_METHOD = "extDict_libNew";
+    static constexpr inline auto EXT_DICT_LIB_CLONE_METHOD = "extDict_libClone";
```

Comment:
Looks like need to mark this PR with `backward incompatible` label.

Reply:
Good question. The "method" names are only used for server-to-bridge communication. This can hardly be considered to be part of ClickHouse's public interface, and unless a user upgrades only the server but not the bridge (which I think is unsupported but I'll ask in Slack), then the renamed method names should not have any impact.

Reply:
I agree but thought at least worth to mention in the change log. (I remember I saw one case in some Yandex prod, when gracefully stopping ch server did not also stop the bridge, while it should have, and only manually killing the process helped)

Reply:
Alright, that makes sense. Will add a note for the change log. Thanks!